Cargo.lock

```diff
@@ -277,7 +277,7 @@ dependencies = [
  "security-framework",
  "security-framework-sys",
  "webpki-root-certs",
- "windows-sys 0.52.0",
+ "windows-sys 0.59.0",
```

Comment:
I think this slipped in accidentally under a `cargo update`. It needs to stay on `0.52` so the minimum version testing covers both.

Reply:
Yeah, it keeps flipping for me. Will revert...